py-polars/tests/unit/io/test_parquet.py

```diff
@@ -136,8 +151,7 @@ def test_parquet_datetime() -> None:
     df = pl.DataFrame(data)
     df = df.with_column(df["datetime"].cast(pl.Datetime))
 
-    # todo! test all compressions here
-    df.write_parquet(f, use_pyarrow=True, compression="snappy")
+    df.write_parquet(f, use_pyarrow=True, compression=compression)
```

Comment:
I think we should check `use_pyarrow=False`. I don't really understand why we test pyarrow here, must have slipped it in somehow.

Reply:
Done via 79f0adb1fb35bf0484cc1c5783ca962356cf307b.